django/core/management/commands/loaddata.py

```diff
@@ -20,15 +20,15 @@
 )
 from django.utils.functional import cached_property
 
+has_bz2 = True
```

Comment:
This format can be more clear:
```python
try:
    import bz2
except ImportError:
    has_bz2 = False
else:
    has_bz2 = True
```

Reply:
I personally prefer what's currently there. If a committer/Django member prefers @pauloxnet's form, I'll change it.